rose/server/main.py

```diff
@@ -14,6 +15,22 @@
 
 def main():
     logging.basicConfig(level=logging.INFO, format=config.logger_format)
+    parser = argparse.ArgumentParser(description="ROSE Server")
+    parser.add_argument("--track_definition", "-t", dest="track_definition",
+                        default="random", choices=["random", "same"],
+                        help="Definition of driver tracks: random or same."
+                             "If not specified, random will be used.")
+
+    args = parser.parse_args()
+    """
+    If the argument is 'same', the track will generate the obstacles in the same place for both drivers.
+    Otherwise, the obstacles will be genrated in random locations for each driver.
+    """
+    if args.track_definition=="same":
```

Comment:
please add spaces before and after the ==

Reply:
Fixed